### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by SteveGoldstein** [AFIDSI/GitPractices#3] [General]
I've created a GitPractices repo, and I included a "Discussions" feature.  Seems like a good feature to try out.

**Comment by ilmcconnell** [comment]
@SteveGoldstein Try @-ing some others!

**Comment by sstevens2** [comment]
Ooo interesting!  I've never used a discussions section before!  I usually have discussions in Issues.  This might be a nice separation.
